### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by AvidItalianCoffeeDrinker456** [moseoridev/WarpShare#18]
The app on some devices dosent see if permissions have been granted 

I download and Installed the app and it asked me to grant permission like nearby device sharing and others but when I granted them nothing happened I'm going to keep trying

Please add a skip permissions button or fix the actual bug

**Comment by Zrapata** [comment]
I could fix this issue by turning on `NerbyShare` and  turning off `Power Saving Mode` at least on my Galaxy S10e

**Comment by Tomcat014** [comment]
Yeah i have the same issue on Blackshark 5

**Comment by rom4ster** [comment]
Yeah please fix this , its essentially just UI saying "you suck, no app for you"
I hate this, I HATE THIS!!!!
